app/src/main/kotlin/com/mapbox/vision/teaser/OnBackPressedListener.kt

```diff
@@ -2,5 +2,5 @@ package com.mapbox.vision.teaser
 
 interface OnBackPressedListener {
 
-    fun onBackPressed(): Boolean
+    fun onBackPressed() = false
```

Comment:
Is it a placeholder? Could we just remove it? Or am I missing smth?

Reply:
fixed, this default = false and usage was a bit confusing